docker-compose.prod.yaml

```diff
@@ -189,6 +189,7 @@ services:
       - MONGO_REPLACE_DATA_INSIDE_TRANSACTION=false
       - KAKAROT_ADDRESS=0x11c5faab8a76b3caff6e243b8d13059a7fb723a0ca12bbaadde95fb9e501bda
       - DEFAULT_BLOCK_GAS_LIMIT=7000000
+      - WHITE_LISTED_EIP_155_TRANSACTION_HASHES=
```

Comment:
I think it's better to remove this, let's just index all of them

Reply:
fixed